lib/hexpm_web/views/api/web_auth_view.ex

```diff
@@ -0,0 +1,14 @@
+defmodule HexpmWeb.API.WebAuthView do
+  use HexpmWeb, :view
+
+  def render("code.json", reponse) do
+    Map.take(reponse, [:device_code, :user_code])
+  end
+
+  def render("access.json", %{write_key: write_key, read_key: read_key}) do
+    %{
+      write_key: Routes.api_key_url(Endpoint, :show, write_key),
+      read_key: Routes.api_key_url(Endpoint, :show, read_key)
```

Comment:
The `Key.user_secret` needs to be returned here since it's generated once and never stored.